pkgs/sketch_pad/firebase.json

```
@@ -7,6 +7,33 @@
       "**/.*",
       "**/node_modules/**"
     ],
+    "redirects": [
+      {
+        "source": "/embed-dart.html",
+        "destination": "/?embed=true",
+        "type": 301
+      },
+      {
+        "source": "/embed-flutter.html",
+        "destination": "/?embed=true",
+        "type": 301
+      },
+      {
+        "source": "/embed-inline.html",
+        "destination": "/?embed=true",
+        "type": 301
+      },
+      {
+        "source": "/embed-flutter_showcase.html",
+        "destination": "/?embed=true",
+        "type": 301
+      },
+      {
+        "source": "/embed-inline.html",
+        "destination": "/?embed=true",
+        "type": 301
+      }
```

Looks like this entry is duplicated. Did you mean a different file? Or can just remove:

```suggestion
      }
```

---

The original issue had listed `dart`, `flutter`, `html`, `inline`, and `showcase`; I had only included a few as redirects. Perhaps the the 2nd `inline` is meant to be `html`? cc @johnpryan

---

Ah, you're right this is meant to be `html`. https://github.com/dart-lang/dart-pad/pull/2800/commits/9ab54b7ec84699f236e8bfab699944c192a24867 should fix it.